pysnooper/tracer.py

```diff
@@ -307,6 +315,13 @@ def __exit__(self, exc_type, exc_value, exc_traceback):
         self.target_frames.discard(calling_frame)
         self.frame_to_local_reprs.pop(calling_frame, None)
 
+        start_time = self.start_times.pop(-1)
+        duration = datetime_module.datetime.now() - start_time
+        now_string = pycompat.timedelta_isoformat(duration, timespec='microseconds')
```

Comment:
```suggestion
        now_string = pycompat.timedelta_isoformat(duration)
```

No point in passing the default value.

Reply:
Delete default value.